src/components/BookForm.js

```diff
@@ -0,0 +1,12 @@
+import { Form } from 'react-router-dom';
+import Input from './Input';
+
+export default function BookForm() {
+  return (
+
+    <Form>
+      <Input name="title" />
+      <Input name="author" />
+    </Form>
```

Comment:
- [ ] Please kindly add the **`add button`** to submit a book according to the design. 👍 This will give your project a good start-up you can build upon in the subsequent milestone. ℹ️ 

> Kindly take  note that styling is not necessary for this milestone. ℹ️ 

## Template
<img width="819" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/221375590-0080144a-0b85-4159-bb34-643756b80882.png">